### DISCUSSION TITLE
Has the paper for this project been accepted？

### DISCUSSION THREAD

**Discussion by HushNino** [BAAI-Agents/Cradle#34] [General]
I saw the title of this paper on the ICLR website. Has this paper been accepted by ICLR 2024 or other journals？

**Comment by WeihaoTan** [comment]
Thanks for reaching out. The old paper (only including RDR2) is accepted by the ICLR LLM-agent workshop, rather than the main conference. The incoming version of the paper (with 4 games and various software) is still under review.
